synapse/models/science.py

```diff
@@ -0,0 +1,92 @@
+import synapse.lib.module as s_module
+
+class ScienceModule(s_module.CoreModule):
+
+    def getModelDefs(self):
+        return (('sci', {
+            'types': (
+                ('sci:hypothesis:type:taxonomy', {}, {}),
+                ('sci:hypothesis', ('guid', {}), {
+                    'doc': 'A hypothesis or theory.'}),
```

Comment:
Recommend changing definition to not include the word "hypothesis". Alternative wording could be something like "A testable theory or supposition".

Reply:
Actually come to think of it, this may be a silly comment